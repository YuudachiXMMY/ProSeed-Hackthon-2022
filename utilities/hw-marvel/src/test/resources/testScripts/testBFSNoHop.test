# Test find a path from a node to itself

CreateGraph graph1

AddNode graph1 node1

FindPath graph1 node1 node1

