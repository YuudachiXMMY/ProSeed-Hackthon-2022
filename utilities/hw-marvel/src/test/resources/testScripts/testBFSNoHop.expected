# Test find a path from a node to itself

created graph graph1

added node node1 to graph1

path from node1 to node1: